lib/graphiti/adapters/abstract.rb

```diff
@@ -255,6 +255,10 @@ def paginate(scope, current_page, per_page)
         raise "you must override #paginate in an adapter subclass"
       end
 
+      def cursor_paginate(scope, current_page, per_page)
```

Comment:
Nitpick: arguments should probably be called `after` and `size`?

Reply:
Whoops, yep good catch - force-pushed 👍